parallel-install/pkg/preinstaller/resourcemanager.go

```diff
@@ -38,8 +37,13 @@ type DefaultResourceManager struct {
 }
 
 // NewResourceManager creates a new instance of ResourceManager.
-func NewDefaultResourceManager(kubeconfigPath string, log logger.Interface, retryOptions []retry.Option) (*DefaultResourceManager, error) {
-	restConfig, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+func NewDefaultResourceManager(kubeconfigPath, kubeconfigRaw string, log logger.Interface, retryOptions []retry.Option) (*DefaultResourceManager, error) {
```

Comment:
Why don't we just pass directly`*rest.Config` as a parameter?

Reply:
Currently all objects handle `rest.Config` on their own and create it internally.